flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/planner/codegen/calls/BridgingFunctionGenUtil.scala

```diff
@@ -147,7 +146,21 @@ object BridgingFunctionGenUtil {
         externalOperands,
         returnType)
     } else {
-      generateScalarFunctionCall(ctx, functionTerm, externalOperands, outputDataType)
+      val targetOperands = externalOperands.zip(operands).map{
+        case(targetOperand, sourceOperand) => {
+          if (sourceOperand.isInstanceOf[ExternalGeneratedExpression]) {
+            val expression = sourceOperand.asInstanceOf[ExternalGeneratedExpression]
+            ExternalGeneratedExpression.fromGeneratedExpression(
```

Comment:
Why do this?

Reply:
ExternalGeneratedExpression is not a case class, so the method copy of parent class GeneratedExpression create a new object is the instance of  GeneratedExpression not ExternalGeneratedExpression

Reply:
I have rollback and override GeneratedExpression copy method in ExternalGeneratedExpression